sys/include/net/ng_ipv6/netif.h

```diff
@@ -84,6 +84,15 @@ extern "C" {
  */
 
 /**
+ * @{
+ * @name Flags for the interfaces
```

Comment:
Because flags have a width in memory that is in most cases smaller than `sizeof(enum)` (most bit fields I know of are 16 bits max, on most of our newer platforms, `sizeof(enum)` is however 32 bits). This results in every assignment needed to be cast to either `uint8_t` or `uint16_t`. With macros you don't need to cast since they are typeless. 

Making the `enum` packed makes it's width unpredictable in terms of alignment issues, when used in struct (which is not the case here, I know).